deepchecks/core/condition.py

```diff
@@ -58,6 +58,7 @@ class ConditionCategory(enum.Enum):
 
     FAIL = 'FAIL'
     WARN = 'WARN'
+    PASS = 'PASS'
```

Comment:
when doing inside condition function `return ConditionResult(...)` we will be able to define the category as "passed" + we have the is_pass boolean. Isn't that confusing?

Reply:
Fixed it + fixed confusing category when condition is passed (used to be failed when passed)